### DISCUSSION TITLE
Translate Folders in backend?

### DISCUSSION THREAD

**Discussion by r00key** [pimcore/pimcore#11228] [Q&A / Support]
Hello,

Is a way to translate the folders and the object names in the backend?

I know how to use the localized fields for making the content of an object multi-language. I know how to make the class names multi language using the shared translations and admin translations. But how do I make the names in the trees  in the backend multi language?

**Comment by passioneight** [comment]
Localizing the keys in the tree is not really something that is supported out of the box, afaik.

Chances are, you could somehow misuse [admin styles](https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Custom_Icons.html), but additionally to performance issues, you'll also need to find a way to support renaming the elements. Also, what if my user's language is English and I want to rename an element for people in Germany? So you see, there's plenty to consider.

If you _must_ have some form of localization, maybe you can use English as key and show the translated value as tooltip. In which case you'll have to create translation-keys for every translated element, which might bloat the translations. Also, the performance issue still persists.

Personally, I'd expect the users to know their work field well enough to be able to read the english version of the related words, otherwise how would the communicate with their colleagues?
